hack/osbuild2copr.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+ROOTDIR=$(dirname $0)
+
+REPO_MODE=
+case "$1" in
+copr)
+    REPO_MODE=enable
+    ;;
+appstream)
+    REPO_MODE=disable
+    ;;
+*)
+    echo "Usage: $(basename $0) <copr | appstream>"
+    exit 1
+    ;;
+esac
+
+echo "Removing the existing 'osbuild' packages..."
+LIST2REMOVE=$(rpm -qa | egrep 'osbuild|cockpit-composer|rpm-ostree' || true)
+[ ! -z "${LIST2REMOVE}" ] && sudo rpm -e ${LIST2REMOVE}
+
+echo "Configuring the 'copr' repositories..."
+sudo dnf copr -y $REPO_MODE @osbuild/osbuild
+sudo dnf copr -y $REPO_MODE @osbuild/osbuild-composer
+
+# sudo dnf copr -y $REPO_MODE walters/ostreerhel8
```

Comment:
Note that "dnf copr" command does not work for walters/ostreerhel8 because it only contains CentOS8 (not RHEL8) builds. They are compatible, but dnf refuses to use them.